RealmSwift/SwiftUI.swift

```diff
@@ -1729,6 +1729,34 @@ private class ObservableAsyncOpenStorage: ObservableObject {
         storage = ObservableAsyncOpenStorage(asyncOpenKind: .asyncOpen, app: app, configuration: configuration, partitionValue: nil)
     }
 
+    /**
+     Initialize the property wrapper for a partition sync based sync app.
+     - parameter appId: The unique identifier of your Realm app, if empty or `nil` will try to retrieve latest singular cached app.
+     - parameter partitionValue: The `BSON` value the Realm is partitioned on.
+     - parameter timeout: The maximum number of milliseconds to allow for a connection to
+                 become fully established., if empty or `nil` no connection timeout is set.
+     */
+    public init<Partition>(appId: String? = nil,
+                           partitionValue: Partition,
+                           timeout: UInt? = nil) where Partition: BSON {
+        let app = ObservableAsyncOpenStorage.configureApp(appId: appId, timeout: timeout)
+        // Store property wrapper values on the storage
+        storage = ObservableAsyncOpenStorage(asyncOpenKind: .asyncOpen, app: app, configuration: nil, partitionValue: AnyBSON(partitionValue))
+    }
+
+    /**
+     Initialize the property wrapper for a flexible sync app.
+     - parameter appId: The unique identifier of your Realm app, if empty or `nil` will try to retrieve latest singular cached app.
+     - parameter timeout: The maximum number of milliseconds to allow for a connection to
+                 become fully established., if empty or `nil` no connection timeout is set.
+     */
+    public init(appId: String? = nil,
+                timeout: UInt? = nil) {
+        let app = ObservableAsyncOpenStorage.configureApp(appId: appId, timeout: timeout)
+        // Store property wrapper values on the storage
+        storage = ObservableAsyncOpenStorage(asyncOpenKind: .asyncOpen, app: app, configuration: nil, partitionValue: nil)
```

Comment:
If the configuration is `nil` how does it know about the `initialSubscriptions`?